cmd/kritis/admission/main.go

```diff
@@ -65,8 +65,39 @@ func main() {
 		fmt.Println(version.Commit)
 		return
 	}
+
+	kritisConfigs, err := kritisconfig.KritisConfigs("")
+	if err != nil {
+		errMsg := fmt.Sprintf("error getting kritis config: %v", err)
+		glog.Errorf(errMsg)
+		return
+	}
+
+	metadataBackend := DefaultMetadataBackend
+	cronInterval := DefaultCronInterval
+	serverAddr := DefaultServerAddr
+
+	if len(kritisConfigs) == 0 {
+		glog.Errorf("No KritisConfigs found in any namespace, will assume the defaults")
+	} else {
+		if len(kritisConfigs) > 1 {
+			glog.Errorf("More than 1 KritisConfig found, will use the 0th object")
```

Comment:
WDYT of returning an error here instead to enforce one KritisConfig/cluster?

Reply:
Good question. I've considered this and at the time decided that it was better to parse any config than failing the program at this point. However, given that the CRD is per-cluster, I can see the value in the argument that this is an unexpected condition. Updated the condition to return an error, open to feedback from others if there are valid cases to have more than 1 KritisConfig defined.